packages/flutter/lib/src/material/input.dart

```diff
@@ -119,7 +127,8 @@ class _InputFieldState extends State<InputField> {
     ];
```

Comment:
This the comment that appears here. Maybe it's not clear?
```
// Since the focusKey may have been created here, defer building the
// RawInput until the focusKey's context has been set. This is necessary
// because the RawInput will check the focus, like Focus.at(focusContext),
// when it builds.
```